images/telegraf/Dockerfile

```diff
@@ -12,21 +12,11 @@ ENV DEBIAN_FRONTEND=noninteractive \
 RUN apt-get update && \
```

Comment:
Do you think we can prefix all `RUN` commands with `set -ex` as it makes the output much better for debugging (i.e. by exposing the value of environment variables, etc.)? I'd want to do this universally across all our Dockerfile unless there's a drawback I'm not aware of.